docs/guide-authentication.md

```diff
@@ -58,17 +58,90 @@ Once enabled the sign in dialog will display additional sign in options:
 Selecting e.g. Google will redirect the browser to their website so the user can authenticate there and return to MOLGENIS once authentication has successfully completed.
 The permissions you have once authenticated are the default user permissions set by an administrator.   
 
-### Configuring single sign-on   
+## Configuring single sign-on   
```

Comment:
NOTE: only works if on server ntp is installed and running.

Reply:
I've added a little explanation of that at the end